src/figma/tokens.json

```diff
@@ -329,232 +329,232 @@
         "default": {
           "value": "#24272A",
-          "description": "(grey800: #24272A) icon.default should be used for all icons used as CTAs that are not a primary action on background.default or background.alternative",
+          "description": "(grey800: #24272A) For default neutral icons.",
           "type": "color"
         },
         "muted": {
           "value": "#BBC0C5",
-          "description": "(grey200: #BBC0C5) icon.muted should be used for all low priority icon used on background.default or background.alternative. These could also include placeholder or inactive icons. It will also not meet AAA or AA accessibility standards.",
+          "description": "(grey200: #BBC0C5) For inactive or low priority icons.",
           "type": "color"
         }
       },
       "border": {
         "default": {
           "value": "#BBC0C5",
-          "description": "(grey200: #BBC0C5) border.default should be used for all neutral component border colors such as cards or inputs in default state.",
+          "description": "(grey200: #BBC0C5) For default neutral borders with visible contrast. (Example: text inputs)",
           "type": "color"
         },
         "muted": {
           "value": "#D6D9DC",
-          "description": "(grey100: #D6D9DC) border.muted should be used for any borders with low contrast or subtle definition like dividers",
+          "description": "(grey100: #D6D9DC) Weaker contrast option for neutral borders. (Example: dividers)",
           "type": "color"
         }
       },
       "overlay": {
         "default": {
           "value": "#00000099",
-          "description": "(black000: #000000 60% opacity) overlay.default should be used for all general overlay backgrounds",
+          "description": "(black000: #000000 60% opacity) For shading layers behind modality screens.",
           "type": "color"
         },
         "alternative": {
           "value": "#000000CC",
-          "description": "(black000: #000000 80% opacity) overlay.alternative should be used for all general overlay backgrounds that need a higher contrast than overlay.default",
+          "description": "(black000: #000000 80% opacity) Added contrast option of overlay.",
           "type": "color"
         },
         "inverse": {
           "value": "#FCFCFC",
-          "description": "(white010: #FCFCFC) overlay.inverse should be used only as the foreground element on top of overlay.default used for text or icons",
+          "description": "(white010: #FCFCFC) [Deprecated] overlay.inverse should be used only as the foreground element on top of overlay.default used for text or icons",
           "type": "color"
         }
       },
       "primary": {
         "default": {
           "value": "#037DD6",
-          "description": "(blue500: #037DD6) primary.default should be used for all primary action components such as buttons or links.",
+          "description": "(blue500: #037DD6) For interactive or active semantic",
```

Comment:
@Akatori-Design If we are using this for all active semantics should the check icon when and account is active be blue? Also there are many "interactive" components that don't use blue? icon/default, danger/critical buttons etc

Reply:
Sorry, missed this comment.  My logic was,  for colors with clear purpose I stated  "For (purpose)".  For color options  relative to the default options (stronger, weaker...etc), I stated "(adjective) option". Were the descriptions confusing you?

Reply:
- You're right about the active state. It should be blue if it's communicating "selected" state, not green cuz green is like success or positive. 
- Yea Im not exactly sure on the primary description. How would you describe it? "For primary user action related elements"?

Reply:
Yeah I think "For primary user action related elements" is more specific. I can make some suggestions and you can commit if you like.

Reply:
Sounds good.